GDJS/Runtime/pixi-renderers/pixi-json-manager.ts

```diff
@@ -111,24 +142,15 @@ namespace gdjs {
      * @param resourceName The resource pointing to the json file to load.
      * @param callback The callback function called when json is loaded (or an error occurred).
      */
-    loadJson(resourceName: string, callback: JsonManagerRequestCallback): void {
+    load(resourceName: string, callback: JsonManagerRequestCallback): void {
       const resource = this._resources.get(resourceName);
       if (!resource) {
-        callback(
-          new Error(
-            'Can\'t find resource with name: "' +
-              resourceName +
-              '" (or is not a json resource).'
-          ),
-          null
-        );
-        return;
+        return callback(new Error(`Can't find resource with name: "${resourceName}" (or is not a json resource).`), null);
       }
 
       // Don't fetch again an object that is already in memory
       if (this._loadedJsons[resourceName]) {
-        callback(null, this._loadedJsons[resourceName]);
-        return;
+        return callback(null, this._loadedJsons[resourceName]);
       }
       // Don't fetch again an object that is already being fetched.
       {
```

Comment:
Odd block {} ?

Reply:
was moved from json-manager. dont understand the reason. removed